src/detours.cpp

```diff
@@ -1959,12 +2040,303 @@ LONG WINAPI DetourUpdateThread(_In_ HANDLE hThread)
     }
 
     t->hThread = hThread;
+	t->fCloseThreadHandleOnDestroyDetourThreadObject = fCloseThreadHandleOnDestroyDetourThreadObject;
     t->pNext = s_pPendingThreads;
     s_pPendingThreads = t;
 
     return NO_ERROR;
 }
 
+//Code modified from https://github.com/apriorit/mhook/blob/master/mhook-lib/mhook.c
```

Comment:
The source project is also released under the MIT license, so it should not conflict with this project. If you are still worried, I can write the same function code again in my own way.

Reply:
The MIT license requires complete attribution, by also including the copyright noticec and a copy of the license file. So I can understand why Detours would want to avoid that.

Reply:
OK, I have refine the code to remove use the code from mhook.

Reply:
thanks